deployments/development-install/helmfile.yaml

```diff
@@ -0,0 +1,188 @@
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: storage-system-jwt-secret
+            type: Opaque
+            data:
+              storage-system-jwt-secret: {{ randAlphaNum 30 | b64enc | quote }}
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: storage-system
+            type: Opaque
+            data:
+              api-key: {{ randAlphaNum 30 | b64enc | quote }}
+              user-id: {{ uuidv4 | b64enc | quote }}
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: transfer-secret
+            type: Opaque
+            data:
+              transfer-secret: {{ randAlphaNum 30 | b64enc | quote }}
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: thumbnails-transfer-secret
+            type: Opaque
+            data:
+              thumbnails-transfer-secret: {{ randAlphaNum 30 | b64enc | quote }}
+
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: ldap-bind-secrets
+            type: Opaque
+            data:
+              reva-ldap-bind-password: {{ randAlphaNum 30 | b64enc | quote }}
+              idp-ldap-bind-password: {{ randAlphaNum 30 | b64enc | quote }}
+              graph-ldap-bind-password: {{ randAlphaNum 30 | b64enc | quote }}
+
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: ldap-ca
+            type: Opaque
+            {{ $ca := genCA "ldap-ca" 365 -}}
+            data:
+              ldap-ca.crt: {{ $ca.Cert | b64enc | quote }}
+
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: ldap-cert
+            type: Opaque
+            {{ $lcapCert := genSignedCert "idm" nil (list "idm") 365 $ca -}}
+            data:
+              ldap.key: {{ $lcapCert.Key | b64enc | quote }}
+              ldap.crt: {{ $lcapCert.Cert | b64enc | quote }}
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: admin-user
+            type: Opaque
+            data:
+              user-id: {{ uuidv4 | b64enc | quote }}
+              password: {{ randAlphaNum 10 | b64enc | quote }}
+
+          - |
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: idp-secrets
+            type: Opaque
+            {{ $privKey := genPrivateKey "rsa" }}
+            data:
+              encryption.key: {{ randAscii 32 | b64enc | quote }}
+              private-key.pem: {{ $privKey | b64enc | quote }}
+          - |
+            apiVersion: v1
+            kind: ConfigMap
+            metadata:
+              name: storage-users
+            data:
+              storage-uuid: 1284d238-aa92-42ce-bdc4-0b0000009157
```

Comment:
Maybe generate the uuid with `uuidv4`?

Reply:
Yeah, I didn't touch it because it wasn't a secret, but having it generated will be nicer.